releases/data-center/syskit-point-2025-2/point-2025-2-78-release-note.md

```diff
@@ -0,0 +1,125 @@
+---
+description: This article lists improvements and bug fixes in Syskit Point version 2025.2.78
+---
+
+# Syskit Point 2025.2.78
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think.](https://www.syskit.com/company/contact-us/)
+
+**Product version:** 2025.2.78
+
+**Build number:** 46
```

Comment:
build # i date će se mijenjati.
našli smo bugove pa release čeka dok se to ne fixa. a kad se fixa, bit će i novi build.